src/lib/Telemetry/telemetry.cpp

```diff
@@ -204,120 +204,85 @@ bool Telemetry::RXhandleUARTin(uint8_t data)
     return true;
 }
 
-bool Telemetry::AppendTelemetryPackage(uint8_t *package)
+void Telemetry::AppendTelemetryPackage(uint8_t *package)
 {
-    const crsf_header_t *header = (crsf_header_t *) package;
+    const crsf_ext_header_t *header = (crsf_ext_header_t *) package;
```

Comment:
Only messages in the frame range 0x28-0x96 have ext headers and the classic tlm frames are not ext frames. While nothing bad happens in this code with this cast it's not always correct. That is the reason why I had the if (header->type >= CRSF_FRAMETYPE_DEVICE_PING) before I did interpret it as ext frame.